### DISCUSSION TITLE
[HxSideBar] I cant get the collapse to work

### DISCUSSION THREAD

**Discussion by checkman123** [havit/Havit.Blazor#763] [General]
Hi,
I followed this [example ](https://havit.blazor.eu/components/HxSidebar#responsive-layout-sample), but I couldn't get the collapse to work for the sidebar.
I have both the CSS and bootstrap JS in the head and body correctly, too.

Not sure what am I doing wrong here

**Comment by crdo** [comment]
@checkman123 Need more information, screenshots, or reproducing repository in order to be able to help you.

**Comment by checkman123** [comment]
That's mb. Here is my [repo](https://github.com/checkman123/MooPing)
When I clicked on the tiny button to collapse the sidebar.. it doesn't collapse like the example. 
I may have missed something? hmm

![image](https://github.com/havit/Havit.Blazor/assets/27747471/dedb5aa4-9046-4c6a-b2fe-b2b191123676)

**Reply by alphaleonis** [reply]
@checkman123 Did you ever find a solution to this? Just trying out Havit.Blazor for the first time here, and encountered the same problem.

**Reply by alphaleonis** [reply]
What fixed this problem for me was adding `@rendermode="InteractiveServer"` to both the `<Routes>` and `<HeadOutlet>` elements in my App.razor in a .NET 8 Blazor server app.

**Reply by checkman123** [reply]
yup, sorry for the late reply. I was busy yesterday and forgot to reply. Good to know that you fixed the problem! :D
